### DISCUSSION TITLE
Can we add new status such as warning in the allure report using python behave?

### DISCUSSION THREAD

**Discussion by Soundarya-Sundaram** [allure-framework/allure2#2204] [Questions & Support]
In allure report, the possible status are Passed, Failed, Broken, Skipped and Unknown. So, can we add a new status such as Warning where i want the step to not pass or fail but highlight as warning status. Is it possible to customize the status in allure report?

**Comment by er-anupamkumar** [comment]
I guess this will be very helpful to have. I came across this topic as I want to have a warning in my report with a specific condition is met. But I could not find any feature like that.
